modules/serverless-workflow-traffic-splitting-flags.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+//serverless/knative-client.adoc
+
+[id="traffice-splitting-flags_{context}"]
+= Traffic splitting flags 
+
+`kn` supports traffic operations on the traffic block of a service as part of the `kn service update` command.
+
+The following table displays a summary of traffic splitting flags, value formats, and the operation the flag performs. The *Repetition* column denotes whether repeating the particular value of flag is allowed in a `kn service update` command.
```

Comment:
For "\*Repetition\*" - asterisks are pretty much reserved for GUI items. Suggest to just use backticks or quotes around it.